crates/jarscanner/src/main.rs

```diff
@@ -0,0 +1,47 @@
+use clap::Parser;
+use std::collections::HashMap;
+use std::error::Error;
+use std::path::PathBuf;
+
+use bzl_gen_jarscanner as jarscanner;
+
+#[derive(Parser)]
+#[command(author, version, about)]
+#[command(version = "0.1.0")]
+#[command(about = "Jar to containing classes tool", long_about = None)]
+struct Opt {
+    #[arg(long)]
+    out: String,
+
+    #[arg(long)]
+    label: String,
+
+    #[arg(long)]
+    input_jar: String,
+
+    #[arg(long)]
+    relative_path: String,
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error>> {
+    let opt = Opt::parse();
+    let output_path = PathBuf::from(opt.out);
+    let input_jar_path = PathBuf::from(opt.input_jar);
+
+    let label_to_allowed_prefixes = HashMap::new();
+    // Insert any prefixes you want to allow here. For example:
```

Comment:
Looking for feedback here: we could make this a CLI argument, or have it read in a JSON file, or?